UPGRADE-5.3.md

```diff
@@ -63,6 +63,12 @@ HttpKernel
  * Deprecate `HttpKernelInterface::MASTER_REQUEST` and add `HttpKernelInterface::MAIN_REQUEST` as replacement
  * Deprecate `KernelEvent::isMasterRequest()` and add `isMainRequest()` as replacement
 
+Intl
+----------
+
+ * Deprecate `DateFormatter` namespace, use symfony/polyfill-intl-icu ^1.21 instead
```

Comment:
These are internal classes. What should be mentioned are the polyfill classes for PHP class names